### PR TITLE
Full Text Search

### PR DESCRIPTION
**Pull request by povodok**
#2

### CODE REVIEW

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <ul class="nav navbar-nav navbar-right">
       <form class="navbar-form navbar-left" role="search" class="search_form" style="margin-top:6px;margin-bottom:4px;">
         <div class="form-group">
-          <input type="text" class="form-control input-sm" placeholder="Пошук" size="30"/>
+          <input name ="q" type="text" class="form-control input-sm" placeholder="Пошук" size="30"/>
```
Comment (mahasamatman) [inline app/views/layouts/_navigation.html.erb]: There are plenty of Rails' helpers to create inputs, please use `text_field_tag` helper to render this input.

**Review by mahasamatman — CHANGES_REQUESTED** [review]
Please add requested changes.

File: app/views/layouts/_navigation.html.erb (hunk 1 above)
Comment (shhavel) [reply app/views/layouts/_navigation.html.erb]: Please ensure that input `value`  equals `params[:q]`.
So when user types a query into input and press button `Search` after page reload input should contain the value of entered by user query.

Comment (shhavel) [reply app/views/layouts/_navigation.html.erb]: Also please use `I18n` for `"Пошук"`

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -12,9 +12,9 @@
     <ul class="nav navbar-nav navbar-right">
       <form class="navbar-form navbar-left" role="search" class="search_form" style="margin-top:6px;margin-bottom:4px;">
         <div class="form-group">
-          <input name ="q" type="text" class="form-control input-sm" placeholder="Пошук" size="30"/>
+          <%= text_field_tag 'q', nil, size: 30, class: 'form-control input-sm', value: params[:q] %>
```
Comment (mahasamatman) [inline app/views/layouts/_navigation.html.erb]: 1. Use symbol as param name: `text_field_tag :q`
2. Second param is the value. No need to set it explicitly with `value: params[:q]`, just use `text_field_tag :q, params[:q], size: 30, class: 'form-control input-sm'`

File: app/views/layouts/_navigation.html.erb [hunk 3]
```diff
@@ -12,9 +12,9 @@
...
         </div>
-        <input type="submit" class="btn btn-default btn-sm" value="Пошук" />
+        <%= submit_tag t('button') ,  class: "btn btn-default btn-sm", name: nil  %>
```
Comment (mahasamatman) [inline app/views/layouts/_navigation.html.erb]: Please give that button appropriate name. There will be a lot of buttons. Maybe something like `t('button_search_top')` is good.

**Review by mahasamatman — CHANGES_REQUESTED** [review]

File: app/views/layouts/_navigation.html.erb (hunk 3 above)
Comment (shhavel) [reply app/views/layouts/_navigation.html.erb]: I think just `search` is ok for the translation key.
Also please remove space before comma, and live only one space before `%>`

File: app/views/layouts/_navigation.html.erb [hunk 4]
```diff
@@ -12,9 +12,9 @@
     <ul class="nav navbar-nav navbar-right">
       <form class="navbar-form navbar-left" role="search" class="search_form" style="margin-top:6px;margin-bottom:4px;">
         <div class="form-group">
-          <input type="text" class="form-control input-sm" placeholder="Пошук" size="30"/>
+          <%= text_field_tag :q, params[:q], size: 30, class: 'form-control input-sm' %>
         </div>
-        <input type="submit" class="btn btn-default btn-sm" value="Пошук" />
+        <%= submit_tag t('button_search'),  class: "btn btn-default btn-sm", name: nil %>
```
Comment (shhavel) [inline app/views/layouts/_navigation.html.erb]: Please remove one space before `class: ...`

**Review by mahasamatman — APPROVED** [review]

**Review by shhavel — APPROVED** [review]

**Pull request merged**